reading-data/reading_data_solution.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""reading_data
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1YKoHc9bkg7ghdlA_n00X1ckbQ_PSgBjO
+
+# **mtcars.csv**
+"""
+
+import numpy as np
+import pandas as pd
+
+mtcars = pd.read_csv("/content/mtcars.csv")
+mtcars
+
+mtcars_new = mtcars.to_csv("/content/mtcars2.csv") # Choose a new name for the document inside the arugment
+
+mtcars_new_read = pd.read_csv("/content/mtcars2.csv")
+mtcars_new_read
```

Comment:
Need to compare new dataframe to original dataframe that was initially read (applies to all files). Hint: use equals() method on dataframe object.